src/batch/src/error.rs

```diff
@@ -200,3 +186,13 @@ impl From<ConnectorError> for BatchError {
         Self::Connector(value.into())
     }
 }
+
+// Define a external system error
+def_anyhow_variant! {
+    pub BatchExternalSystemError,
+    BatchError ExternalSystemError,
+
+    tokio_postgres::Error => "Postgres Batch Query Error",
```

Comment:
We may omit `Batch Query` as such error must be returned to the client when user issuing a batch query, so we and users know it's a "batch query error".

```suggestion
    tokio_postgres::Error => "Postgres error",
```

Reply:
Fixed

Reply:
https://github.com/risingwavelabs/risingwave/pull/18982/commits/02b28665aa19ec7433871a0524f6bd7052d4bee0